include/mxnet/lib_api.h

```diff
@@ -214,6 +214,18 @@ enum MXDType {
   kUNSET = 100,
 };
 
+/*
+ * MXTensor storage type.
+ */
+enum MXStorageType {
+  // dense
```

Comment:
can we add a "kUnknownStorage = -1" entry too? and get rid of the "-1" magic numbers everywhere?